.bazelci/presubmit.yml

```diff
@@ -316,17 +316,45 @@ tasks:
     working_directory: examples/multi_python_versions
     platform: windows
 
+  integration_test_custom_py_project_ubuntu_min_workspace:
```

Comment:
This will deplete the number of runners we have available for running builds. I remember @rickeylev did some work to optimize this so maybe relying on the bazel integration test framework would be better here?